endpoints/signup.go

```diff
@@ -28,6 +28,15 @@ func (c *Client) Signup(req types.SignupRequest) (*types.SignupResponse, error)
 		return nil, err
 	}
 
+	if req.ConfirmationRedirectUrl != "" {
```

Comment:
I would expect to see updates to the integration tests to exercise this new branch and verify the response includes the expected redirect before this can be merged